specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2019-08-09-preview/vmwarevirtustream.json

```diff
@@ -702,6 +750,25 @@
     }
   },
   "definitions": {
+    "QuotaGet": {
+      "description": "Subscription quotas",
+      "properties": {
+        "hostsRemaining": {
+          "description": "Remaining hosts quota by sku type",
+          "type": "object",
+          "readOnly": true,
+          "additionalProperties": {
+            "type": "integer",
+            "readOnly": true
+          }
+        },
+        "quotaEnabled": {
+          "description": "Host quota is active for current subscription",
```

Comment:
>subscription [](start = 59, length = 12)

this is also not accurate for a tenant level API

Reply:
you are correct, this was a confusion on my part.  Updated this pr and the manifest to remove the Tenant routing.